redbot/cogs/trivia/session.py

```diff
@@ -207,7 +208,8 @@ async def wait_for_answer(self, answers, delay: float, timeout: float):
             )
         except asyncio.TimeoutError:
             if time.time() - self._last_response >= timeout:
-                await self.ctx.send(_("Guys...? Well, I guess I'll stop then."))
+                with contextlib.suppress(discord.HTTPException):
+                    await self.ctx.send(_("Guys...? Well, I guess I'll stop then."))
```

Comment:
Could you send the traceback that you are getting here that isn't a Forbidden nor NotFound?

Reply:
Yeah I don't know why I have set HTTPException here, just changed.